<!doctype html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta name="generator" content="TiddlyWiki" />
<meta name="tiddlywiki-version" content="5.1.12" />
<meta name="viewport" content="width=device-width, initial-scale=1.0" />
<meta name="apple-mobile-web-app-capable" content="yes" />
<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
<meta name="mobile-web-app-capable" content="yes"/>
<meta name="format-detection" content="telephone=no">
<link id="faviconLink" rel="shortcut icon" href="favicon.ico">
<link rel="stylesheet" href="static.css">
<title>The structure of the genotype-phenotype map strongly constraints the evolution of non-coding RNA: Cosmos — a non-linear personal web notebook</title>
</head>
<body class="tc-body">

<section class="tc-story-river">

<p><div class="tc-tiddler-frame tc-tiddler-view-frame tc-tiddler-exists   "><div class="tc-tiddler-title">
<div class="tc-titlebar">
<span class="tc-tiddler-controls">
<span class=" tc-reveal"><button aria-label="more" class="tc-btn-invisible" title="More actions">


</button><div class=" tc-reveal" hidden="true"></div></span><span class=" tc-reveal" hidden="true"></span><span class=" tc-reveal" hidden="true"></span><span class=" tc-reveal" hidden="true"></span><span class=" tc-reveal" hidden="true"></span><span class=" tc-reveal" hidden="true"></span><span class=" tc-reveal"><button aria-label="edit" class="tc-btn-invisible" title="Edit this tiddler">


</button></span><span class=" tc-reveal" hidden="true"></span><span class=" tc-reveal" hidden="true"></span><span class=" tc-reveal" hidden="true"></span><span class=" tc-reveal" hidden="true"></span><span class=" tc-reveal" hidden="true"></span><span class=" tc-reveal"><button aria-label="close" class="tc-btn-invisible" title="Close this tiddler">


</button></span><span class=" tc-reveal" hidden="true"></span><span class=" tc-reveal" hidden="true"></span><span class=" tc-reveal" hidden="true"></span>
</span>

<span>

<span class="tc-tiddler-title-icon" style="fill:;">

</span>



<h2 class="tc-title">
The structure of the genotype-phenotype map strongly constraints the evolution of non-coding RNA
</h2>

</span>

</div>

<div class="tc-tiddler-info tc-popup-handle tc-reveal" hidden="true"></div>
</div><div class=" tc-reveal" hidden="true"></div>
<div class=" tc-reveal">
<div class="tc-subtitle">
<a class="tc-tiddlylink tc-tiddlylink-missing" href="guillefix.html">
guillefix
</a> 23rd April 2016 at 12:28am
</div>
</div>
<div class=" tc-reveal">
<div class="tc-tags-wrapper"></div>
</div>
<div class="tc-tiddler-body tc-reveal"><p>See <a class="tc-tiddlylink tc-tiddlylink-resolves" href="MMathPhys%2520oral%2520presentation.html">MMathPhys oral presentation</a></p><p><a class="tc-tiddlylink-external" href="http://rsfs.royalsocietypublishing.org/content/5/6/20150053.full" rel="noopener noreferrer" target="_blank">The structure of the genotype–phenotype map strongly constrains the evolution of non-coding RNA</a></p><p><strong>Non-coding RNA</strong> (ncRNA) is RNA whose function is not to encode information. It's function may then be structural, or catalytic for instance, and is most often determined by its secondary structure, which is then the phenotype of interest.</p><p><b>The distribution of properties found in ncRNA in nature (from <a class="tc-tiddlylink-external" href="http://www.ncbi.nlm.nih.gov/pubmed/17099231" rel="noopener noreferrer" target="_blank">fRNAdb database</a>) closely follows that obtained by <em>G-sampling</em></b> (uniform sampling over genotypes). Due to the bias in the GP map, this sampling is very different from <em>P-sampling</em> (uniform sampling over phenotypes). The strong bias makes certain structures appear much more often, which has been called <em>convergent evolution</em> in <a class="tc-tiddlylink tc-tiddlylink-resolves" href="Evolution.html">Evolution</a> (part of the general phenomenon of <em><a class="tc-tiddlylink tc-tiddlylink-resolves" href="Homoplasy.html">homoplasy</a></em>). An example is the ubiquity of the <em>hammerhead ribozyme</em> through all the kingdoms of life.</p><p><img src="http://d2gme0e5d9kd75.cloudfront.net/content/royfocus/5/6/20150053/F2.large.jpg" width="500"></p><p><small>Figure 2.
Comparison of P-sampled and G-sampled distributions to natural data for L = 20 RNA. The P-sampled PP(Ω) (red diamonds) measures the probability distribution for a phenotype to have a given NS size Ω. It differs markedly from G-sampled PG(Ω) (blue circles), generated by random sampling over genotypes. Error bars arise from binning data. The black and cyan lines are theoretical approximations to PP(Ω) and PG(Ω), respectively (see Methods). The probability distribution of Ω for the SSs all 7327 (non-trivial) L = 20 sequences for Drosophila melanogaster from the fRNAdb database [21] (green squares) is much closer to the G-sampled PG(Ω) than to the P-sampled PP(Ω). Inset: all 11 218 SS phenotypes (purple triangles) ranked by NS size Ω. There is strong bias, just 5% of phenotypes take up 58% of all genotypes. The 7327 natural data points (green squares) are clustered at lower rank (larger Ω). (Online version in colour.)</small></p><p>The <b>number of 'relevant structures'</b> can be estimated by the entropy <span><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>H</mi></mrow><annotation encoding="application/x-tex">H</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="strut" style="height:0.68333em;"></span><span class="strut bottom" style="height:0.68333em;vertical-align:0em;"></span><span class="base textstyle uncramped"><span class="mord mathit" style="margin-right:0.08125em;">H</span></span></span></span></span> of the G-sampled distribution of features (for instance belonging to a certain binned interval of neutral space size, or number of stacks (sets of contiguous base-pairs)), as <span><span class="katex"><span class="katex-mathml"><math><semantics><mrow><msup><mn>2</mn><mi>H</mi></msup></mrow><annotation encoding="application/x-tex">2^H</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="strut" style="height:0.8413309999999999em;"></span><span class="strut bottom" style="height:0.8413309999999999em;vertical-align:0em;"></span><span class="base textstyle uncramped"><span class="mord"><span class="mord mathrm">2</span><span class="vlist"><span style="top:-0.363em;margin-right:0.05em;"><span class="fontsize-ensurer reset-size5 size5"><span style="font-size:0em;">​</span></span><span class="reset-textstyle scriptstyle uncramped"><span class="mord mathit" style="margin-right:0.08125em;">H</span></span></span><span class="baseline-fix"><span class="fontsize-ensurer reset-size5 size5"><span style="font-size:0em;">​</span></span>​</span></span></span></span></span></span></span>. One can define the <em>bias ratio</em> as the ratio of <span><span class="katex"><span class="katex-mathml"><math><semantics><mrow><msup><mn>2</mn><mi>H</mi></msup></mrow><annotation encoding="application/x-tex">2^H</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="strut" style="height:0.8413309999999999em;"></span><span class="strut bottom" style="height:0.8413309999999999em;vertical-align:0em;"></span><span class="base textstyle uncramped"><span class="mord"><span class="mord mathrm">2</span><span class="vlist"><span style="top:-0.363em;margin-right:0.05em;"><span class="fontsize-ensurer reset-size5 size5"><span style="font-size:0em;">​</span></span><span class="reset-textstyle scriptstyle uncramped"><span class="mord mathit" style="margin-right:0.08125em;">H</span></span></span><span class="baseline-fix"><span class="fontsize-ensurer reset-size5 size5"><span style="font-size:0em;">​</span></span>​</span></span></span></span></span></span></span> to the total number of phenotypes.</p><p>Within these relevant structures which <em>arrive</em> during evolution, <b>natural selection still acts</b>, and can be seen for example in the higher stability of natural RNAs vs random G-sampled RNAs. We find that the natural RNAs have slightly more bonds than in G-sampled structures. The bias towards larger <span><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi mathvariant="normal">Ω</mi></mrow><annotation encoding="application/x-tex">\Omega</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="strut" style="height:0.68333em;"></span><span class="strut bottom" style="height:0.68333em;vertical-align:0em;"></span><span class="base textstyle uncramped"><span class="mord mathrm">Ω</span></span></span></span></span> also leads to structures with <b>larger mutational robustness</b> (see <a class="tc-tiddlylink-external" href="http://press.princeton.edu/titles/8002.html" rel="noopener noreferrer" target="_blank">Robustness and Evolvability in Living Systems</a> and <a class="tc-tiddlylink-external" href="http://www.ncbi.nlm.nih.gov/pubmed/7517565" rel="noopener noreferrer" target="_blank">From sequences to shapes and back: a case study in RNA secondary structures</a>).  Larger
robustness is considered to be advantageous [6], so that, in
this important way, phenotype bias facilitates evolution.
The high robustness, however is found both in G and P-sampling because of the high genetic correlations (genes tend to be close in the mutational network to other genes that produce the same phenotype). The genetic correlations are high enough to produce giant connected components (see <a class="tc-tiddlylink-external" href="http://www.nature.com/nature/journal/v225/n5232/abs/225563a0.html" rel="noopener noreferrer" target="_blank">Natural Selection and the Concept of a Protein Space</a>).</p><p>&quot;Bias
means that it will be difficult for evolution to find L ¼ 55 structures
with a large number of stacks, again raising the question
of <u>what kind of functionality is possible in principle that cannot
be reached by evolution because of such phenotype bias
constraints?</u>&quot;</p><p><small>Understanding tip: The line in figure 4 is flat when there are a lot of phenotypes because there are a lot of phenotypes with the same <span><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi mathvariant="normal">Ω</mi></mrow><annotation encoding="application/x-tex">\Omega</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="strut" style="height:0.68333em;"></span><span class="strut bottom" style="height:0.68333em;vertical-align:0em;"></span><span class="base textstyle uncramped"><span class="mord mathrm">Ω</span></span></span></span></span>, and the phenotypes are equally spaced in <span><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>x</mi></mrow><annotation encoding="application/x-tex">x</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="strut" style="height:0.43056em;"></span><span class="strut bottom" style="height:0.43056em;vertical-align:0em;"></span><span class="base textstyle uncramped"><span class="mord mathit">x</span></span></span></span></span> axis in rank plot.</small></p><p>The results that G-sampling produce the same results as the database indicate that some property similar to ergodicity may be at play. G-sampling is an ensemble average, and the database shows a kind of time-average over evolutionary trajectories. However, the process cannot be totally ergodic because <u>evolution is a nonequilibrium process</u>, and effects like long waiting times and the <a class="tc-tiddlylink tc-tiddlylink-resolves" href="Arrival%2520of%2520the%2520frequent.html">Arrival of the frequent</a> are examples of non-ergodic non-equilibrium effects.</p><p>The GP map bias is an example of biases in development or other internal
processes could strongly affect evolutionary outcomes. These have been controversial; however, RNA SS provides perhaps the clearest and most unambiguous
evidence for the importance of bias in shaping
evolutionary outcomes.</p><p>See <a class="tc-tiddlylink tc-tiddlylink-resolves" href="Homoplasy.html">Homoplasy</a> for discussion on the relation to convergent and parallel evolution. Our ability to make detailed predictions about evolutionary
outcomes as well as counterfactuals for RNA may also
shed light on Mayr’s famous distinction between<b> proximate
and ultimate causes</b> in biology (See <a class="tc-tiddlylink-external" href="http://isites.harvard.edu/fs/docs/icb.topic100452.files/Mayr_E._1961._Science_V134.pp1501-1506.pdf" rel="noopener noreferrer" target="_blank"> Cause and effect in biology</a> and <a class="tc-tiddlylink-external" href="https://en.wikipedia.org/wiki/Proximate_and_ultimate_causation" rel="noopener noreferrer" target="_blank">Proximate and ultimate causation</a>). Not sure about this, or if I understand it..</p><p>The GP mapping
constraint has some resemblance to classical morphogenetic
constraints which also bias the arrival of variation [47]. But it also differs, because the latter are conceptualized at the level of
phenotypes and developmental processes, and may have been
shaped by prior selection, whereas the former constraint is a
fundamental property of the mapping from genotypes to phenotypes
and was not selected for (<b>except perhaps at the origin
of life itself </b> <i aria-hidden="true" class="fa fa-exclamation"></i> <i aria-hidden="true" class="fa fa-arrow-right"></i> Still, maybe most possible GP maps have this property anyways (see <a class="tc-tiddlylink tc-tiddlylink-resolves" href="Evolving%2520automata.html">experiments with transducers</a>)).</p><p>Finally, strong phenotype bias is also found in:</p><ul><li>model GP maps for protein tertiary (<a class="tc-tiddlylink-external" href="http://cqb.pku.edu.cn/tanglab/pdf/1996-34.pdf" rel="noopener noreferrer" target="_blank">Emergence of Preferred Structures in a Simple Model of Protein Folding</a>, <a class="tc-tiddlylink-external" href="http://www.englandlab.com/uploads/7/8/0/3/7803054/2003prl.pdf" rel="noopener noreferrer" target="_blank">Structural determinant of protein designability</a>), </li><li>quaternary structure. Self-assembling polyominoes model (<a class="tc-tiddlylink-external" href="http://rsif.royalsocietypublishing.org/content/11/95/20140249" rel="noopener noreferrer" target="_blank">A tractable genotype–phenotype map modelling the self-assembly of protein quaternary structure</a>), </li><li><a class="tc-tiddlylink tc-tiddlylink-resolves" href="Gene%2520regulatory%2520networks.html">Gene regulatory networks</a> (<a class="tc-tiddlylink-external" href="http://www.pnas.org/content/103/11/4180.abstract" rel="noopener noreferrer" target="_blank"> Highly designable phenotypes and mutational buffers emerge from a systematic mapping between network topology and dynamic output</a>, <a class="tc-tiddlylink-external" href="http://www.ncbi.nlm.nih.gov/pubmed/21225054" rel="noopener noreferrer" target="_blank">Evolvability and robustness in a complex signalling circuit</a>)</li><li>and development (<a class="tc-tiddlylink-external" href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000202" rel="noopener noreferrer" target="_blank">An End to Endless Forms: Epistasis, Phenotype Distribution Bias, and Nonuniform Evolution</a>)</li></ul><p>suggesting that some of the results discussed in this paper for
RNA may hold more widely in biology</p><p><img src="img/Selection_177.png"></p><p>See also <a class="tc-tiddlylink tc-tiddlylink-resolves" href="Evolving%2520automata.html">Evolving automata</a></p><p>Paper with several examples of GP maps, including cellular automata map: <a class="tc-tiddlylink-external" href="http://ieeexplore.ieee.org/xpl/login.jsp?tp=&amp;arnumber=870337&amp;url=http%3A%2F%2Fieeexplore.ieee.org%2Fxpls%2Fabs_all.jsp%3Farnumber%3D870337" rel="noopener noreferrer" target="_blank">An investigation of redundant genotype-phenotype mappings and their role in evolutionary search</a></p><table><tbody><tr class="evenRow"><td>It would be interesting to devise <b>artificial methods</b> to search for such undiscovered ribozymes (those that are very improbable for evolution to find), some of which may be more fit than those that Nature has found.</td></tr></tbody></table><p>For this see:</p><p><a class="tc-tiddlylink-external" href="http://nar.oxfordjournals.org/content/31/11/2926.full" rel="noopener noreferrer" target="_blank">Exploring the repertoire of RNA secondary motifs using graph theory; implications for RNA design</a>. tree graphs to describe RNA tree motifs and more general (dual) graphs to describe both RNA tree and pseudoknot motifs. our graph theory approach to RNA structures has implications for RNA genomics, structure analysis and design.</p><p><a class="tc-tiddlylink-external" href="http://www.sciencedirect.com/science/article/pii/S1367593114001215" rel="noopener noreferrer" target="_blank">Experimental fitness landscapes to understand the molecular evolution of RNA-based life</a>
In evolutionary biology, the relationship between genotype and Darwinian fitness is known as a fitness landscape. These landscapes underlie natural selection, so understanding them would greatly improve quantitative prediction of evolutionary outcomes, guiding the development of synthetic living systems. However, the structure of fitness landscapes is essentially unknown. Our ability to experimentally probe these landscapes is physically limited by the number of different sequences that can be identified. This number has increased dramatically in the last several years, leading to qualitatively new investigations. Several approaches to illuminate fitness landscapes are possible, ranging from tight focus on a single peak to random speckling or even comprehensive coverage of an entire landscape. We discuss recent experimental studies of fitness landscapes, with a special focus on functional RNA, an important system for both synthetic cells and the origin of life.</p><hr><p><u>Methods</u></p></div>



</div>

</p>

</section>
</body>
</html>
